src/lib/slider/slider.ts

```diff
@@ -254,6 +266,46 @@ export class MdSlider implements ControlValueAccessor {
     this.onTouched();
   }
 
+  _onKeydown(event: KeyboardEvent) {
+    if (this.disabled) { return; }
+
+    switch (event.keyCode) {
+      case PAGE_UP:
+        this._increment(10);
+        break;
+      case PAGE_DOWN:
+        this._increment(-10);
+        break;
+      case END:
+        this.value = this.max;
+        break;
+      case HOME:
+        this.value = this.min;
+        break;
+      case LEFT_ARROW:
+        this._increment(-1);
+        break;
+      case UP_ARROW:
+        this._increment(1);
+        break;
+      case RIGHT_ARROW:
+        this._increment(1);
+        break;
+      case DOWN_ARROW:
+        this._increment(-1);
+        break;
+      default:
+        return;
+    }
+
+    event.preventDefault();
```

Comment:
Should we only `preventDefault` when it's one of the keys we care about? E.g., pressing spacebar should probably still scroll the page.